src/USER-CONP/compute_conp_vector.cpp

```diff
@@ -55,28 +55,18 @@ ComputeConpVector::ComputeConpVector(LAMMPS *lmp, int narg, char **arg)
   gaussians = 1;
   recalc_every = 0;
   overwrite = 1;
-  assigned = false;
 
   g_ewald = 0.0;
 
-  // get jgroup; igroup defined in parent class
-
-  int n = strlen(arg[3]) + 1;
-  group2 = new char[n];
-  strcpy(group2, arg[3]);
-
-  jgroup = group->find(group2);
-  if (jgroup == -1)
-    error->all(FLERR, "Compute coul/vector group ID does not exist");
-  jgroupbit = group->bitmask[jgroup];
+  assigned = false;
 
   // TODO recalculate coulomb vector every recalc_every
 
-  recalc_every = utils::inumeric(FLERR, arg[4], false, lmp);
+  recalc_every = utils::inumeric(FLERR, arg[3], false, lmp);
```

Comment:
recalc_every kann auch hier komplett raus. Siehe mein Kommentar dazu in compute_matrix.